app/_hub/kong-inc/acme/0.2.11.md

```diff
@@ -216,6 +224,21 @@ $ curl https://mydomain.com
 # Now gives you a valid Let's Encrypt certicate
 ```
 
+#### Renew certificates
+
+The plugin runs a daily check and automatically renews all certificate that
+will expire in less than configured `renew_threshold_days`. The renewal process
+continues on error of individual certificate and retry will be performed the next
+day. Note the renewal config is stored in configured storage backend.
```

Comment:
```suggestion
will expire in less than configured `renew_threshold_days`. If the renewal of an 
individual individual certificate throws an error, the plugin will move on to the next
certificate. It will try renewing all certificates, including those that have previously failed, once per day. Note the renewal config is stored in configured storage backend.
```